bridge/slack/slack.go

```diff
@@ -281,8 +281,14 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 		return "", nil
 	}
 
-	// Handle message deletions.
 	var handled bool
```

Comment:
Please move this between the comment and the `if` statement. No need to create extra empty lines for nothing.

Reply:
I feel this logically belongs outside, as it sets up a var to be used in many places below (it's not logically tied to that bit the comment refers to)